services/argus/client.go

```diff
@@ -80,7 +80,12 @@ func NewAPIClient(opts ...config.ConfigurationOption) (*APIClient, error) {
 		return nil, fmt.Errorf("setting up authentication: %w", err)
 	}
 
-	cfg.HTTPClient.Transport = authRoundTripper
+	roundTripper := authRoundTripper
+	if cfg.Middleware != nil {
+		roundTripper = config.ChainMiddleware(roundTripper, cfg.Middleware...)
```

Comment:
Is this method already released in a new version of `core`?

Reply:
And the `core` dependency updated for all these services?

Reply:
I assume the lint is not failing because it uses the local version of the core? We should actually fix this to have some way that in the pipeline it would use the version that is imported, otherwise we could easily be merging this without noticing and it would be a runtime error

Reply:
Core is now updated and go.mods are using the latest version